doc/api/net.md

```diff
@@ -331,10 +331,18 @@ Returns `server`.
 added: v0.3.4
 -->
 
-This object is an abstraction of a TCP or local socket.  `net.Socket`
-instances implement a duplex Stream interface.  They can be created by the
-user and used as a client (with [`connect()`][]) or they can be created by Node.js
-and passed to the user through the `'connection'` event of a server.
+This class is an abstraction of a TCP socket, a UNIX domain socket, or
+a Windows named pipe. A `net.Socket` is also a [duplex stream][], so it
```

Comment:
Er, wait, I think IPC still works because technically the name IPC doesn't imply it's local, it can be remote. `s/streaming socket/streaming IPC/`?